src/elements/content-preview/ContentPreview.js

```diff
@@ -742,6 +743,11 @@ class ContentPreview extends React.PureComponent<Props, State> {
             fileOpts[fileId].fileVersionId = selectedVersion.id;
         }
 
+        if (activeAnnotationId) {
+            fileOpts[fileId] = fileOpts[fileId] || {};
+            fileOpts[fileId].annotations = { activeId: activeAnnotationId };
```

Comment:
What happens if this property is defined and passed in outside of the element? Should we try to merge into it or is overwriting what we want.

Reply:
I guess I would lean toward overwriting it since it should be the more "correct" `activeId` since it's derived from the passed in `history` location pathname